src/game_components/VotingPage.js

```diff
@@ -0,0 +1,31 @@
+import { Input, Button } from "antd";
+import "antd/dist/antd.css";
+import React, { useState, useEffect } from "react";
+import database from "../database.js";
+
+const VotingPage = (props) => {
```

Comment:
this is stylistic, but I tend to reserve `Page` as a suffix for actual different URLs. Because of that, Pages shouldn't really have any props other than URL params. Perhaps this could be `VotingScreen` instead